rss_test.go

```diff
@@ -0,0 +1,31 @@
+package rssemitter_test
+
+import (
+	"testing"
+
+	"github.com/Bottleneck/rssemitter"
```

Comment:
Please follow standard package naming/referencing

change to: github.com/BottleneckStudio/rssemitter

Reply:
Thanks @kenkoii noted on this one :+1: